Car_Code/cheme_car_datalogging_codebase/cheme_car_datalogging_codebase.ino

```diff
@@ -1,15 +1,63 @@
 // TODO:
 // Serial Interface - Needs to be initialized, look into plotting data with the data streamer add-on in Excel.
-// Temeperature Sensor - Needs to be initialized, periodically poll sensor, store temperature in variable, print out to serial monitor, look into kalman filters for reducing sensor noise.
+// Rida -In Progress- Temperature Sensor - Needs to be initialized, periodically poll sensor, store temperature in a variable, print out to serial monitor, look into Kalman filters for reducing sensor noise.
 // IMU - Needs to be initialized, periodically poll sensor, store Z-axis angle in variable, take initial value as 0 deg print out to serial monitor, look into kalman filters for reducing sensor noise.
 // Stir Bar Motor - Needs to be initialized, just constantly turn at 80% speed for now, we'll tune speed later.
 // Servo Motor - Needs to be initialized, turn 180 deg clockwise once at start to dump reactants, wait 1 sec to finish dumping, turn back 180 deg counter-clockwise to return to upright position, set speed to 100% for now, tune later.
 // MicroSD Card - Store all variable data to MicroSD Card on microcontroller, allow for each run to be saved under a different filename (potentially in CSV format) so we can keep track of everything.
 
+#include <OneWire.h>
+#include <DallasTemperature.h>
+#define ONE_WIRE_BUS 23 // pin for the DS18B20 data line
```

Comment:
Change this from 23 to the new pin assignments (should be A1). Don't forget when you do this to pull changes from the main branch to your branch too so they're both up-to-date as I made some changes to the main branch for the pcb.